### PR TITLE
add unleash logo

### PR DESCRIPTION
**Pull request by haakoneriksen**
Add unleash logo to unleash square

### CODE REVIEW

**Review by ivarconr — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -77,6 +77,7 @@
                 </div>
                 <div class="grid__unit">
                     <div style="min-height:50px;width:100%;padding:16px;background:#CCFFEC">
+                        <img src="unleash-logo.png" align="left" alt="Unleash logo" style="padding-right:30px">
```
Comment (SimenB) [inline index.html:80]: svg? point to some cdn-version instead of hard-coding here?

Comment (haakoneriksen) [reply index.html:80]: Couldn't find an SVG version

Comment (haakoneriksen) [reply index.html:80]: BTW, this is hosted at github.io, I'm pretty sure that's as good CDN as any

Comment (SimenB) [reply index.html:80]: It was more to avoid keeping a copy in this repo, and always get the latest. no issue, though

**Review by theneva — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (theneva) [reply index.html:80]: Much better than nothing IMO!

**Pull request merged**